scapy/arch/bpf/core.py

```diff
@@ -108,7 +135,9 @@ def get_dev_bpf():
         try:
             fd = os.open("/dev/bpf%i" % bpf, os.O_RDWR)
             return (fd, bpf)
-        except OSError:
+        except OSError as ex:
+            if ex.errno == 13:  # Permission denied
+                raise Scapy_Exception("Permission denied")
```

Comment:
TODO extend message. e.g.

Permission denied: could not open /dev/bpf%i. Make sure you are running Scapy using sudo !

This should help us avoiding the thousands of "It doesn't work, it says /dev/bpf" issues

Reply:
Sounds good.